Pod/Classes/Controllers/SoundCloudLoginWebViewController.m

```diff
@@ -16,7 +16,7 @@ @interface SoundCloudLoginWebViewController () <UIWebViewDelegate>
 @implementation SoundCloudLoginWebViewController
 
 + (UINavigationController *)instantiateWithLoginURL:(NSString *)loginURL redirectURL:(NSString *)redirectURL resultBlock:(void (^)(BOOL result, NSString *code))resultBlock {
-    UINavigationController *navController = [[UIStoryboard storyboardWithName:@"SoundCloudLogin" bundle:nil] instantiateInitialViewController];
+    UINavigationController *navController = [[UIStoryboard storyboardWithName:@"SoundCloudLogin" bundle:[NSBundle bundleForClass:self]] instantiateInitialViewController];
```

Comment:
This is exactly the same, only shorter, since `(self == [self class])` in class methods.